Sync_app/konturmarket/konturmarket_class_lib.py

```diff
@@ -123,3 +125,43 @@ def sync_assortment(self) -> bool:
             return km_models.KonturMarketDBGood.save_objects_to_db(list_km_goods=km_goods)
 
         return True
+
+    def create_sales_journal(self, date_: datetime.date) -> bool:
+        """Создание журнала розничных продаж в системе ЕГАИС."""
+        if self.login():
+            url_sales_read: KonturMarketUrl = get_url(UrlType.EGAIS_SALES_JOURNAL_READ)
+            response = self._session.get(
+                url=url_sales_read.url,
+                params={
+                    "date": date_.__str__(),
```

Comment:
str(date) красивее. А вообще стоит вызывать
```suggestion
                    "date": date_.isoformat(),
```